test/addons-napi/test_wrap/binding.gyp

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think we should add this test to test_general instead of using a new addon. Each addon makes the compile just a little bit longer.